wishing_well/util.py

```diff
@@ -75,7 +75,7 @@ def get_cache_path():
     # python cannot do this without raising an error, and neither can the default
     # windows copy command, so we instead delegate this task to powershell's Copy-Item
     try:
-        path = Path(game_path) / 'GenshinImpact_Data/webCaches/2.13.0.1/Cache/Cache_Data/data_2'
```

Comment:
Since it is likely that this is going to change in the future, it would be better if the code automatically picked the correct folder instead of requiring code changes after every patch.

Reply:
I'll submit a fix with this code on the weekend as I already wrote one for warp-journal.